### ISSUE TITLE
Training conditions

### ISSUE THREAD

**Issue by ykk136** [wvangansbeke/Sparse-Depth-Completion#37]
Hi, I'm very grateful you can open your code for this depth completion task. Here I have a question: 
1. Did you implement this training process on a single gpu or multi-gpu?
2. What learning rate decay scheduler did you use? And, did you use the 'scheduler.step' under each epoch or  under 'for i, (inputs, gt) in enumerate(train_loader)'?

Thank you very much!

**Comment by wvangansbeke** [comment]
Hi @ykk136,

- You can run it on multiple gpus if you set `args.multi` to True. (See [this line](https://github.com/wvangansbeke/Sparse-Depth-Completion/blob/master/main.py#L143).)
- I used the `reduce on plateau` scheduler at the end of each epoch. (See [this line](https://github.com/wvangansbeke/Sparse-Depth-Completion/blob/master/main.py#L352).)

Hope this helps.

**Comment by ykk136** [comment]
Thank you, I will check out later. Grateful!
